packages/one-app-hmr/lib/webpack/loaders.js

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2020 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import path from 'path';
+import ExtractCssChunks from 'extract-css-chunks-webpack-plugin';
```

Comment:
[`mini-css-extract-plugin`](https://www.npmjs.com/package/mini-css-extract-plugin) did not work as expected... however `extract-css-chunks-webpack-plugin` did get the job done.